src/ExContext.elm

```diff
@@ -206,6 +208,22 @@ getType =
     getFromContext .types
 
 
+{-| Get's arity of the function in the module
+-}
+getArity : String -> String -> Context -> Maybe Int
+getArity m fn ctx =
+    ctx.modules
+        |> Dict.get m
+        |> Maybe.map (.definitions)
+        |> Maybe.andThen (Dict.get fn)
+        |> Maybe.map (.arity)
+
+
+hasMatchingArity : Context -> String -> String -> List a -> Bool
+hasMatchingArity c mod fn args =
+    List.length args == Maybe.withDefault -1 (getArity mod fn c)
```

Comment:
Given the arguments are the target of the function the name should reflect that.
Either "areMatchingArity" or "matchArity" would probably fit

Reply:
Also a function doc for the sake of being consistent